core-web/apps/dotcms-ui/src/app/portlets/dot-edit-page/seo/components/dot-device-selector-seo/dot-device-selector-seo.component.scss

```diff
@@ -109,7 +109,7 @@ $device-selector-name-margin: 0.375rem;
         }
 
         border-radius: 0.75rem;
```

Comment:
The border-radius don't use any of the vars too? `$border-radius-lg`?

Reply:
We do have vars, we don't have .75rem but we need to talk to Melissa because as my understanding we have the one that she uses in Figma.

Reply:
@fmontes @oidacra It seems like Melissa updated the figma recently, going to use the variable